Makefile

```diff
@@ -25,13 +38,25 @@ deploy: $(TOOLBIN)/kubectl $(TOOLBIN)/helm
 	$(TOOLBIN)/helm install fybrik charts/fybrik --values $(VALUES_FILE) \
                --namespace $(KUBE_NAMESPACE) --wait --timeout 120s
 
+pre-test: generate manifests $(TOOLBIN)/etcd $(TOOLBIN)/kube-apiserver
+	mkdir -p /tmp/taxonomy
+	mkdir -p /tmp/adminconfig
+	cp charts/fybrik/files/taxonomy/*.json /tmp/taxonomy/
+	cp charts/fybrik/files/adminconfig/*.rego /tmp/adminconfig/
+	mkdir -p manager/testdata/unittests/basetaxonomy
+	mkdir -p manager/testdata/unittests/sampletaxonomy
+	cp charts/fybrik/files/taxonomy/*.json manager/testdata/unittests/basetaxonomy
+	cp charts/fybrik/files/taxonomy/*.json manager/testdata/unittests/sampletaxonomy
+	go run main.go taxonomy compile -o manager/testdata/unittests/sampletaxonomy/taxonomy.json \
+  	-b charts/fybrik/files/taxonomy/taxonomy.json \
+		$(shell find samples/taxonomy/example -type f -name '*.yaml')
+	cp manager/testdata/unittests/sampletaxonomy/taxonomy.json /tmp/taxonomy/taxonomy.json
+
 .PHONY: test
 test: export MODULES_NAMESPACE?=fybrik-blueprints
 test: export CONTROLLER_NAMESPACE?=fybrik-system
-test:
-	$(MAKE) -C manager pre-test
+test: pre-test
 	go test -v ./...
```

Comment:
In manager test there were some environment variables defined (e.g. modules namespace).